FreeRTOS/FreeRTOS
# Fix duration of TimerMode Task in WIN32-MingW Demo

Description
-----------
prvDemonstrateChangingTimerReloadMode is a highest priority task which deletes itself after performing the tests. The suicidal tests in death.c uses the number of tasks in the system and for this number to be accurate, suicidal tests must run after the prvDemonstrateChangingTimerReloadMode task has deleted itself. 

After prvDemonstrateChangingTimerReloadMode deletes itself, actual cleanup happens in the IDLE task. This means that the idle task must get a chance to reclaim prvDemonstrateChangingTimerReloadMode task’s resources before the suicidal tests run. 

This commit reduces the duration of prvDemonstrateChangingTimerReloadMode to ensure that the IDLE task runs and reclaims prvDemonstrateChangingTimerReloadMode task’s resources before suicidal tests run.

Test Steps
-----------
```
make FULL_DEMO = 1
qemu-system-arm -machine mps2-an385 -monitor null -semihosting --semihosting-config enable=on,target=native -kernel ./build/RTOSDemo.axf -serial stdio -nographic
```
Before applying the changes
```
Error: Death - tick count 10004 
```
After applying the changes
```
OK: No errors - tick count 10004 
OK: No errors - tick count 20003 
OK: No errors - tick count 30003 
OK: No errors - tick count 40003 
OK: No errors - tick count 50003 
OK: No errors - tick count 60003 
OK: No errors - tick count 70003 
OK: No errors - tick count 80003
```

Checklist:
----------



Related Issue
-----------
#1080 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
